MIGRATE_TO_V2.md

```diff
@@ -0,0 +1,7 @@
+# Migration Guide
```

Comment:
We probably don't want a root level md file for this. Maybe best to create this in `website/src/_posts`